crates/uv-cli/src/lib.rs

```diff
@@ -5554,12 +5554,15 @@ pub struct PythonUninstallArgs {
     /// The Python version(s) to uninstall.
     ///
     /// See `uv help python` to view supported request formats.
-    #[arg(required = true)]
     pub targets: Vec<String>,
 
     /// Uninstall all managed Python versions.
     #[arg(long, conflicts_with("targets"))]
     pub all: bool,
+
+    /// Uninstall outdated Python versions, keeping the latest in each family.
```

Comment:
I think you'll need to be a bit clearer here. Maybe "the latest patch version for each minor version"? and provide an example?